lib/config/config.go

```diff
@@ -75,6 +77,33 @@ func GetRemoteOriginUrl() string {
 	return util.GetCommandOutput("git", "remote", "get-url", "origin")
 }
 
+func GetRemoteOriginHostname() string {
```

Comment:
Given that we already have a `GetRemoteOriginUrl` method, what do you think about simplifying this method to a `GetHostnameOfUrl` method, and providing the URL for which to return the hostname as a parameter? This would make this method better testable.